src/sentry/event_manager.py

```diff
@@ -294,6 +294,31 @@ def get_stored_crashreports(cache_key: str | None, event: Event, max_crashreport
     return query[:max_crashreports].count()
 
 
+def increment_group_tombstone_hit_counter(tombstone_id: int | None, event: Event) -> None:
+    if tombstone_id:
+        try:
+            from sentry.models.grouptombstone import GroupTombstone
+
+            group_tombstone = GroupTombstone.objects.get(id=tombstone_id)
+            buffer_incr(
+                GroupTombstone,
+                {"times_seen": group_tombstone.times_seen + 1 if group_tombstone.times_seen else 1},
```

Comment:
You don't want to include the current count here. Instead only pass `1` here. The buffers system will generate `F(column) + value` when the buffered values are flushed to postgres.